src/DotVVM.Framework/Resources/Scripts/DotVVM.PostBackHandlers.ts

```diff
@@ -15,8 +25,21 @@ class ConfirmPostBackHandler extends DotvvmPostBackHandler {
     }
 }
 
-class DotvvmPostBackHandlers {
-    public confirm = (options: any) => new ConfirmPostBackHandler(<string>options.message);
+class PostbackOptions {
+    constructor(public readonly postbackId: number, public readonly sender?: HTMLElement, public readonly args : any[] = [], public readonly viewModel?: any, public readonly viewModelName?: string, public readonly validationTargetPath?: any) {}
+}
+
+class ConfirmPostBackHandler2 implements DotvvmPostbackHandler2 {
```

Comment:
I think there is no need to have two separated interfaces just because you need to overload some arguments. BTW: @exyi Why do you use types instead of interfaces here?